### ISSUE TITLE
Configure Cloudflare Domain for Web Application and Landing Page

### ISSUE THREAD

**Issue by keinsell** [keinsell/neuronek#448]
There is a `neuronek.xyz` domain on my Cloudflare account and it should be imported to the Terraform, neccessary DNS records for Vercel should be added and viola - there will be some accessibility to the application. The planned layout of domain is:

- `api.neuronek` for the public API
- `app.neuronek` for the web application
- `neuronek.xyz` for the landing page from which user would be able to discover app distributions
